SQL/database_changelog.txt

```diff
@@ -1,15 +1,32 @@
 Any time you make a change to the schema files, remember to increment the database schema version. Generally increment the minor number, major should be reserved for significant changes to the schema. Both values go up to 255.
 
-The latest database version is 5.13; The query to update the schema revision table is:
+The latest database version is 5.14; The query to update the schema revision table is:
 
-INSERT INTO `schema_revision` (`major`, `minor`) VALUES (5, 13);
+INSERT INTO `schema_revision` (`major`, `minor`) VALUES (5, 14);
 or
-INSERT INTO `SS13_schema_revision` (`major`, `minor`) VALUES (5, 13);
+INSERT INTO `SS13_schema_revision` (`major`, `minor`) VALUES (5, 14);
 
 In any query remember to add a prefix to the table names if you use one.
 
 -----------------------------------------------------
 
+Version 5.14, xx May 2021, by Anturke
+Added exploration drone adventure table
+
+```
+DROP TABLE IF EXISTS `text_adventures`;
+CREATE TABLE `text_adventures` (
+	`id` int(11) NOT NULL AUTO_INCREMENT,
+	`adventure_data` LONGTEXT NOT NULL,
+	`uploader` VARCHAR(32) NOT NULL,
+	`timestamp` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+	`approved` BOOLEAN NOT NULL DEFAULT FALSE,
```

Comment:
I know this is real nitpicky but in mariadb `BOOLEAN` is an alias for `TINYINT(1)` so could you use that for consistency.

Reply:
Sure